### PR TITLE
Speechbrain emotion recognition with OpenVINIO

### PR DESCRIPTION
**Pull request by spradeep28**
OpenVINO Jupyter notebook demonstrating SpeechBrain emotion recognition with OpenVINO.

### CODE REVIEW

File: notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb [hunk 1]
```diff
@@ -0,0 +1,454 @@
+{
```
Comment (sbalandi) [inline notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb:1]: <p>I tried to run it and got error:  <code>OSError: libtorch_cuda.so: cannot open shared object file: No such file or directory,</code> Did you face with the same ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2064/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (sbalandi) [inline notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb:1]: <p>Could you please add more information about foreign_class and what happening here , what are CustomEncoderWav2vec2Classifier/ustom_interface.py ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2064/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (sbalandi) [inline notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb:1]: <p>Could you please add more information about output parameters ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2064/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (sbalandi) [inline notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb:1]: <p>Maybe it is good to add some comparison with original model by benchmark_app and some demo with gradio with possibility of trying a custom file ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2064/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (spradeep28) [reply notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb:1]: Tested the notebook before PR. Did not face this error.

Comment (eaidova) [reply notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb:1]: @sbalandi  probably you have torch installed without cuda.
@spradeep28  does this notebook has dependency on cuda? Can we avoid this?

Comment (eaidova) [inline notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    %pip install speechbrain
</pre><p>probably issue with cuda can be from this install as speechbrain has torch in dependencies and by default for linux it installed for cuda. need to be added -extra-index-url <a href="https://download.pytorch.org/whl/cpu" target="_blank">https://download.pytorch.org/whl/cpu</a> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2064/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (aleksandr-mokrov) [inline notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb:1]: <pre class="ql-syntax" spellcheck="false">output_tensor = compiled_model(wavs)[output]
</pre><p>Just </p>

<pre class="ql-syntax" spellcheck="false">output_tensor = compiled_model(wavs)[0]
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2064/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (sbalandi) [reply notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb:1]: I tried this and --upgrade --force-reinstall for torch and it works now , thank you !

Comment (spradeep28) [reply notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb:1]: Addressed

[the remark this answers is not in the document]
Comment (spradeep28) [reply notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb:1]: Addressed

Comment (spradeep28) [reply notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb:1]: tested with --upgrade --force-reinstall working as expected. no cuda error.

Comment (spradeep28) [reply notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb:1]: Will plan it for next release (with gradio).

Comment (spradeep28) [reply notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb:1]: Done

Comment (spradeep28) [reply notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb:1]: Added details.

Comment (spradeep28) [reply notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb:1]: Updated pip install with cpu wheel.

**Review by sbalandi — APPROVED** [review]

**Review by eaidova — APPROVED** [review]

File: .ci/ignore_treon_py38.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
 notebooks/surya-line-level-text-detection/surya-line-level-text-detection.ipynb
-notebooks/stable-diffusion-keras-cv/stable-diffusion-keras-cv.ipynb
\ No newline at end of file
+notebooks/stable-diffusion-keras-cv/stable-diffusion-keras-cv.ipynb
+notebooks/speechbrain-emotion-recognition.ipynb
```
Comment (eaidova) [inline .ci/ignore_treon_py38.txt:3]: you forgot notebook subdirectory in path:
```suggestion
notebooks/speechbrain-emotion-recognition/peechbrain-emotion-recognition.ipynb
```

**Pull request merged**